apps/web/app/api/cron/import/csv/route.ts

```diff
@@ -26,23 +26,19 @@ export async function POST(req: Request) {
 
     if (!id || !url) throw new Error("Missing ID or URL for the import file");
 
-    const map = (row: Record<string, string>) => {
-      const createdAt = mapping.createdAt
-        ? new Date(row[mapping.createdAt])
-        : undefined;
-
+    const mapper = (row: Record<string, string>) => {
       return {
         ...Object.fromEntries(
           Object.entries(mapping).map(([key, value]) => [key, row[value]]),
         ),
-        domain: row[mapping.domain]?.replace(/^https?:\/\//, "").split("/")[0],
+        domain: getPrettyUrl(row[mapping.domain]).split("/")[0],
+        // domain.com/path/to/page => path/to/page
         key:
-          row[mapping.key]
-            ?.replace(/^https?:\/\//, "")
-            .split("/")
-            .at(-1) ?? "",
-        createdAt:
-          createdAt && !isNaN(createdAt.getTime()) ? createdAt : undefined,
+          getPrettyUrl(row[mapping.key]).split("/").slice(1).join("/") ??
```

Comment:
I don't think this will work if the mapped `key` is actually just a key and not the full shortlink.

e.g. `"abcde".split("/").slice(1).join("/")` is `""` instead of `"abcde"`